docs/version0.37.md

```diff
@@ -54,6 +55,13 @@ For more information, see [Supported environments](openj9_support.md).
 
 Linux builds for platforms Linux x86 64-bit, Linux on POWER LE 64-bit, and Linux on IBM Z 64-bit now use gcc 11.2 instead of gcc 10.3. Linux AArch64 64-bit continues to use the gcc 10.3 compiler. See the list of [build environments](openj9_support.md#build-environments).
 
+### Support added for the `com.sun.management.ThreadMXBean` interface
+
+IBM [`ThreadMXbean` interface](https://www.eclipse.org/openj9/docs/api-langmgmt/) (`com.ibm.lang.management.ThreadMXBean`) currently extends [`java.lang.management.ThreadMXBean`](https://docs.oracle.com/javase/8/docs/api/java/lang/management/ThreadMXBean.html) interface. With this release, the IBM `ThreadMXBean` interface extends [`com.sun.management.ThreadMXBean`](https://docs.oracle.com/javase/8/docs/jre/api/management/extension/com/sun/management/ThreadMXBean.html) interface, whereby the IBM `ThreadMXbean` interface now contains all the methods of the `java.lang.management.ThreadMXBean` interface with additional methods from the `com.sun.management.ThreadMXBean` interface.
```

Comment:
This shouldn't use "IBM". The package name includes "ibm" because IBM implemented it originally, but IBM contributed it to Eclipse OpenJ9. It would be incompatible to change the package name. In place of "IBM" pls use OpenJ9.

It should be "The OpenJ9 ...",  "currently extends the ..." and "interface extends the ..."